### DISCUSSION TITLE
直播卡顿问题及弹幕加粗问题

### DISCUSSION THREAD

**Discussion by ybhgl** [ywmoyue/biliuwp-lite#337] [bug]
### Bug 描述

最近几个版本直播为啥看1秒卡5秒，基本无法正常播放，网络没有问题，用逍遥橙子的[聚合直播](https://github.com/xiaoyaocz/AllLive)就可以一直正常播放

从4.6.16一路往回装测试到4.6.7（卸载重装），都有这个问题，再早的版本似乎不能正常播放直播了
~感觉问题是缓冲区有问题？直播卡顿的时候网速不动，要等1-2s才会出现2mb/s左右的网速，然后再过1s左右才会加载出画面~

把直播链接抓出来直接播放也很卡，不知道是什么原因

对比了一下本软件与聚合直播的直播链接，本软件是m3u8，聚合直播是flv

### 操作步骤

```plain text
打开直播间，画质为原画，播放1s卡5s
```

### 期望结果

不卡顿

### 截图

另外发现一个小问题，弹幕引擎选择寒霜弹幕使，图里面的两个设置会消失，但是其实打开了加粗的选项再切回寒霜弹幕使，加粗的效果是会生效的，能否在选择寒霜弹幕使引擎时加入加粗的选项？
![20231004_102004](https://github.com/ywmoyue/biliuwp-lite/assets/35188529/a8e0824f-eef5-4c37-b54e-688532386330)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

Windows 11 21H2 (Build 22000)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
从4.6.7开始默认播放方式改为hls了，那下个版本我加个默认播放方式选项吧

**Comment by ywmoyue** [comment]
加粗这个问题，我之前自测的时候没看出来加粗有生效就给移除了，下个版本加上
